README.md

```diff
@@ -118,3 +118,4 @@ To Contribute to this project, please see the [Contributing Guide](./CONTRIBUTIN
 - Jim Lindeman - Sr. Software Engineer - IBM
 - Lucas Holmquist - Sr. Software Engineer - Red Hat
 - Michael Dawson - Node.js lead for Red Hat and IBM
+- Wojciech Trocki - Software Engineer - Red Hat
```

Comment:
Btw. @mhdawson we do not have this info on website. Website has separate README.md with only introduction part.